test/test_bson_custom_types.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
```

Comment:
Any specific reason for the utf-8 encoding?

Reply:
⌘-C + ⌘-V 😀

Reply:
Removing it.